<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboDialog</class>
 <widget class="QDialog" name="ComboDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">#Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="editKeyword"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelKeyword">
       <property name="text">
        <string>Keyword:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="editSnippet">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="labelSnippet">
       <property name="text">
        <string>Snippet:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOk">
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editKeyword</tabstop>
  <tabstop>editSnippet</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>actionOk</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ComboDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOk</sender>
   <signal>triggered()</signal>
   <receiver>ComboDialog</receiver>
   <slot>onActionOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editKeyword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ComboDialog</receiver>
   <slot>updateGui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSnippet</sender>
   <signal>textChanged()</signal>
   <receiver>ComboDialog</receiver>
   <slot>updateGui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionOk()</slot>
  <slot>updateGui()</slot>
 </slots>
</ui>
